src/data/Navbar.tsx

```
@@ -1,10 +1,17 @@
 import React from "react";
 import {GrActions, GrIntegration, GrProjects} from "react-icons/gr";
 import {FaUserGroup} from "react-icons/fa6";
-import {AiFillProduct, AiOutlineProfile} from "react-icons/ai";
-import {IoSettingsOutline} from "react-icons/io5";
+import {
+    AiFillAlert,
+    AiFillAlipaySquare,
```

ESLint

[error] 'AiFillAlipaySquare' is defined but never used.

(@typescript-eslint/no-unused-vars)